src/Extension/Publishable/PublishableSiteTree.php

```diff
@@ -90,6 +90,27 @@ public function objectsToDelete($context)
      */
     public function urlsToCache()
     {
-        return [Director::absoluteURL($this->getOwner()->Link()) => 0];
+        $link = $this->getOwner()->Link();
+        if (strpos($link, '?') === false) {
+            return [Director::absoluteURL($link) => 0];
+        }
+
+        // Find querystrings
+        $queryargs = [];
+        if (strpos($link, '?') !== false) {
```

Comment:
You don't need this check, you'll only be here if it's true

Reply:
Nice catch 👍